Repositories/toDos.js

```diff
@@ -68,11 +78,24 @@ async function assignTo (body) {
   }
 }
 
+async function getToDosByStatus (status) {
```

Comment:
you don't need to create seperate service for this function. you can embed it to getTodos 
GET /todos?status='TODO'  is status is not available you will return all todos belong to particular user. if status is provided you will filter